.github/workflows/ci.yml

```diff
@@ -237,6 +244,29 @@ jobs:
           if-no-files-found: error
           retention-days: 1
 
+      # Globstar is supported in Bash 4.0+, `macos-latest` runner uses Bash 3.2.
+      - run: find build -name "*.dSYM" -prune -exec mv {} debug/ \;
+        if: ${{ contains('ios macos', matrix.platform) }}
+      - run: |
+          shopt -s globstar
+
+          mkdir debug
+          mv build/web/**/*.map debug/
+          cd build/web && cp --parents **/*.js ../../debug
+        if: ${{ matrix.platform == 'web' }}
```

Comment:
@tyranron, regarding the source map separate artifact uploading, Sentry utilizes `.map` files as well as `.js` sources, here are the logs:

```
Source Map Upload Report
  Scripts
    ~/assets/packages/medea_jason/assets/pkg/medea_jason.js
    ~/firebase-messaging-sw.js
    ~/flutter.js
    ~/flutter_service_worker.js
  Minified Scripts
    ~/assets/packages/media_kit/assets/web/hls1.4.10.js (sourcemap at hls.min.js.map)
    ~/assets/packages/wakelock_plus/assets/no_sleep.js (sourcemap at ../../../../main.dart.js.map)
    ~/canvaskit/canvaskit.js (sourcemap at ../main.dart.js.map)
    ~/canvaskit/chromium/canvaskit.js (sourcemap at ../../main.dart.js.map)
    ~/canvaskit/skwasm.js (sourcemap at ../main.dart.js.map)
    ~/canvaskit/skwasm.worker.js (sourcemap at ../main.dart.js.map)
    ~/main.dart.js (sourcemap at main.dart.js.map)
  Source Maps
    ~/main.dart.js.map
```

Thus `.map` is moved to `debug`, and `.js` files are copied over with `--parents` (there are two `canvaskit.js` files being different).